src/collective/collectionfilter/resources/collectionfilter.js

```diff
@@ -108,6 +110,23 @@ define([
                 this.reloadCollection(collectionURL);
             }.bind(this));
 
+            // OPTION4 - maps filter
+            if (this.$el.hasClass('collectionMaps')) {
+                $('.pat-leaflet', this.$el).on('moveend zoomend', function (e) {
```

Comment:
this event is not fired ... any hints how to get L.map.on('zoomend') here?

Reply:
Good catch :) This should do it: https://github.com/Patternslib/pat-leaflet/pull/3

Reply:
I tried this pull request and it works like a charm! though I need to get somehow `map.getBounds()` of the new view area and I could not find any map object neither on the `original_event` nor on the node ... do you have an idea how to get the new boundaries after zooming or moving?